ts/Core/Renderer/SVG/SVGLabel.ts

```diff
@@ -243,10 +250,11 @@ class SVGLabel extends SVGElement {
     public getBBox(): BBoxObject {
         const bBox = this.bBox;
         const padding = this.padding;
+        const paddingLeft = this.paddingLeft || padding;
```

Comment:
Use `pick()` instead of `||`. Otherwise, when `this.paddingLeft = 0` we will user `this.padding`

Reply:
What would you pass to `attr()` to clear it in this case, some string like `auto` and then check for that in the setter?